server/src/main/java/org/jboss/as/server/deployment/Phase.java

```diff
@@ -380,6 +380,7 @@ public Phase next() {
     public static final int DEPENDENCIES_SEAM                           = 0x0A01;
     public static final int DEPENDENCIES_WS                             = 0x0C00;
     public static final int DEPENDENCIES_SECURITY                       = 0x0C50;
+    public static final int DEPENDENCIES_ELYTRON                        = 0x0C51;
```

Comment:
Perhaps you are remembering David chuckling at peoples' habit of using 0x0A50 as the "halfway" value between 0x0A00 and 0x0B00, instead of 0x0A80. :)